### PR TITLE
.eh_frame: Add $rbp

### PR DESCRIPTION
**Pull request by javierhonduco**
We need the `rbp` offset to CFA to be able to unwind the stack using .eh_frame. I've tried to use the same names as readelf does, so we print `u` when the offset is zero, otherwise, we print the offset (both relative to the CFA).

## Test plan
More testing has to be done (will detail this in an issue soon) but made sure that the output for our CPP test binary is correct. Created to commits so the changes are more easily distinguishable

### CODE REVIEW

**Comment by javierhonduco** [comment]
Copy registers properly rather than just one

File: internal/dwarf/frame/table.go [hunk 1]
```diff
@@ -9,6 +9,9 @@ import (
 	"github.com/parca-dev/parca-agent/internal/dwarf/util"
 )
 
+// TODO(javierhonduco): Add source
+const RBPRegister = 6
```
Comment (kakkoyun) [inline internal/dwarf/frame/table.go:13]: Better to use https://github.com/go-delve/delve/tree/master/pkg/dwarf/regnum

File: pkg/stack/unwind/plan_table.go [hunk 2]
```diff
@@ -243,24 +254,46 @@ func buildTableRows(fde *frame.DescriptionEntry, start uint64) []PlanTableRow {
...
+					row.RA = Instruction{Op: OpUnimplemented}
+				}
+			} else {
 				row.RA = Instruction{Op: OpUnimplemented}
 			}
-		} else {
-			row.RA = Instruction{Op: OpUnimplemented}
+		}
+
+		// Deal with $rbp
```
Comment (kakkoyun) [inline pkg/stack/unwind/plan_table.go:281]: :+1: :heart:

**Review by kakkoyun — APPROVED** [review]
LGTM.

I've added a link for subsequent PRs.

**Comment by javierhonduco** [comment]
Resolve feedback

File: internal/dwarf/frame/table.go (hunk 1 above)
Comment (javierhonduco) [reply internal/dwarf/frame/table.go:13]: Thanks so much! Did not look in the right places

**Pull request merged**